odxtools/table.py

```diff
@@ -132,13 +158,12 @@ def read_table_row_from_odx(et_element):
 
 def read_table_from_odx(et_element):
     """Reads a TABLE."""
-    short_name = et_element.findtext("SHORT-NAME")
-    logger.debug("Parsing TABLE " + short_name)
-
+    short_name = et_element.find("SHORT-NAME").text
```

Comment:
same question as above...

Reply:
same comment as above. please let me know, if you want me to update all the occurrences in this MR?